Fix endpoint URLs for api.equnix.com i.e. add prefix "/metal/v1/"

currently (7.10.2020) 10:44 UTC, the functions exported by Packngo fail like:

```
=======[REQUEST]=============
POST /projects HTTP/1.1
Host: api.equinix.com
User-Agent: packngo/0.4.0
Connection: close
Content-Length: 48
Accept: application/json
Content-Type: application/json
X-Auth-Token: --------------------------------
X-Consumer-Token: packngo test
Accept-Encoding: gzip

{"name":"PACKNGO_TEST_DELME_2d768716_VhRGJIvR"}

2020/10/07 13:43:07 [DEBUG] POST https://api.equinix.com/projects
2020/10/07 13:43:07 
=======[RESPONSE]============
HTTP/1.1 404 Not Found
Connection: close
Content-Length: 164
Content-Security-Policy: frame-ancestors 'none'
Content-Type: application/json
Date: Wed, 07 Oct 2020 10:43:07 GMT
Strict-Transport-Security: max-age=86400
X-Content-Type-Options: nosniff
X-Frame-Options: DENY
X-Xss-Protection: 1

{"fault":{"faultstring":"Unable to identify proxy for host: default and url: \/projects","detail":{"errorcode":"messaging.adaptors.http.flow.ApplicationNotFound"}}}

--- FAIL: TestAccVolumeBasic (0.68s)
    packngo_test.go:93: POST https://api.equinix.com/projects: 404  
FAIL
exit status 1
```

@thebsdbox suggested that endpoints URLs might be updated as:

```
curl -H 'X-Auth-Token: ...' https://api.equinix.com/metal/v1/projects -> is fine
curl -H 'X-Auth-Token: ...' https://api.packet.com/projects -> is fine
curl -H 'X-Auth-Token: ...' https://api.equinix.com/projects -> is broken
```

--- a/packngo.go
+++ b/packngo.go
@@ -324,6 +324,9 @@ type requestDoer interface {
 // NewRequest inits a new http request with the proper headers
 func (c *Client) NewRequest(method, path string, body interface{}) (*retryablehttp.Request, error) {
 	// relative path to append to the endpoint url, no leading slash please
+	if path[0] == '/' {
+		path = path[1:]
+	}
 	rel, err := url.Parse(path)
 	if err != nil {
 		return nil, err

